components/omega/configs/Default.yml

```diff
@@ -25,3 +25,69 @@ Omega:
     ViscDel2: 1.0e3
     VelHyperDiffTendencyEnable: true
     ViscDel4: 1.2e11
+  IOStreams:
+    # InitialState should only be used when starting from scratch
+    # After the simulations initial start, the frequency should be
+    # changed to never so that the initial state file is not read.
+    InitialState:
+      UsePointerFile: false
+      Filename: OmegaMesh.nc
+      Mode: read
+      Precision: double
+      Freq: 1
+      FreqUnits: OnStartup
+      UseStartEnd: false
+      Contents:
+        - Restart
+    # Restarts are used to initialize for all job submissions after the very
+    # first startup job. We use UseStartEnd with a start time just after the
+    # simulation start time so that omega does not attempt to use a restart
+    # for the first startup job.
+    RestartRead:
+      UsePointerFile: true
+      PointerFilename: ocn.pointer
+      Mode: read
+      Precision: double
+      Freq: 1
+      FreqUnits: OnStartup
+      UseStartEnd: true
+      StartTime: 0001-01-01_00:00:01
+      EndTime: 99999-12-31_00:00:00
+      Contents:
+        - Restart
+    RestartWrite:
+      UsePointerFile: true
+      PointerFilename: ocn.pointer
+      Filename: ocn.restart.$Y-$M-$D_$h.$m.$s
+      Mode: write
+      IfExists: replace
+      Precision: double
+      Freq: 6
+      FreqUnits: months
+      UseStartEnd: false
+      Contents:
+        - Restart
+    History:
+      UsePointerFile: false
+      Filename: ocn.hist.$SimTime
+      Mode: write
+      IfExists: replace
+      Precision: double
+      Freq: 1
+      FreqUnits: months
+      UseStartEnd: false
+      Contents:
+        - Tracers
+    Highfreq:
+      UsePointerFile: false
+      Filename: ocn.hifreq.$Y-$M-$D_$h.$m.$s
+      Mode: write
+      IfExists: replace
+      Precision: single
+      Freq: 10
+      FreqUnits: days
+      UseStartEnd: true
+      StartTime: 0001-06-01_00.00.00
+      EndTime: 0001-06-30_00.00.00
```

Comment:
I think we should keep the same time format as in Time Manager?
https://github.com/E3SM-Project/Omega/blob/8b50f3cb3f278cf381cb3a75564cba16f400273e/components/omega/src/infra/TimeMgr.cpp#L3617

Reply:
Streams uses the time manager string-to-time-instant conversion, so colon separator (actually any non-numeric separator) and decimal seconds are supported in a time string. Didn't think the latter was necessary for streams but I can change to the colon separator. I think I inadvertently used a decimal separator here because it's often needed in the filename due to shell interpretation of a colon in many file contexts.